apps/shark_studio/tests/api_test.py

```diff
@@ -7,15 +7,17 @@
 import logging
 import unittest
 from apps.shark_studio.api.llm import LanguageModel
+import gc
 
 
 class LLMAPITest(unittest.TestCase):
-    def testLLMSimple(self):
+    def test01_LLMSmall(self):
```

Comment:
weird naming?

Reply:
forces a certain ordering of tests, but won't be truly necessary until more tests are added.